dspace-api/src/main/java/org/dspace/content/dao/impl/MetadataFieldDAOImpl.java

```diff
@@ -94,27 +97,35 @@ public MetadataField findByElement(Context context, String metadataSchema, Strin
         query.setParameter("name", metadataSchema);
         query.setParameter("element", element);
 
-        if (StringUtils.isNotBlank(qualifier)) {
+        if (qualifier != null) {
```

Comment:
Why are we removing `StringUtils.isNotBlank()` in favor of `!=`?  These aren't the same thing.